patroni/README.md

```diff
@@ -0,0 +1,109 @@
+# Patroni
+Patroni is a template for you to create your own customized, high-availability solution using Python and - for maximum accessibility - a distributed configuration store like ZooKeeper, etcd, Consul or Kubernetes. Database engineers, DBAs, DevOps engineers, and SREs who are looking to quickly deploy HA PostgreSQL in the datacenter-or anywhere else-will hopefully find it useful.
```

Comment:
Please wrap each line after 60 chars or so. Helps to suggest changes to the wording.

In vim you can do this by highlighting the line of text and typing `:gq`